src/router/index.ts

```diff
@@ -31,7 +28,7 @@ const router = new Router({
       component: Policy,
     },
     {
-      path: '/*',
+      path: '/:pathMatch(.*)*',
```

Comment:
Just curious, did the previous path `'/*'` not work?

Reply:
Yeah the old setup doesn't work. For reference, it's from this doc: https://next.router.vuejs.org/guide/essentials/dynamic-matching.html#reacting-to-params-changes